source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc

```diff
@@ -62,10 +63,33 @@ ifdef::local_database_upgrade,remote_database_upgrade[. Log in to the {engine-na
 ----
 # engine-backup --scope=all --mode=backup --file=backup.bck --log=backuplog.log
 ----
-
 . Copy the backup file to a storage device outside of the {virt-product-shortname} environment.
 
 ifdef::SHE_upgrade[]
+
+. Shut down the self-hosted engine.
++
+[options="nowrap" subs="normal"]
+----
+# poweroff
+----
++
+[NOTE]
+====
+Before you shut down the self-hosted engine virtual machine, write down the mac address of the self-hosted engine network interface if you want to reuse it for the {virt-product-fullname} {engine-name} 4.4 deployment.
+====
+
+. Make sure that the self-hosted engine is shut down.
++
+[options="nowrap" subs="normal"]
+----
+# hosted-engine --vm-status | grep 'Engine status|Hostname'
+----
++
+[NOTE]
+====
+If any of the hosts report the `detail` field as `Up`, log in to that specific host and proceed with `hosted-engine --vm-poweroff`.
```

Comment:
```suggestion
If any of the hosts report the `detail` field as `Up`, log in to that specific host and shut it down with the `hosted-engine --vm-shutdown` command.
```
Again...you use a poweroff command, should be shutdown.

Reply:
the command is supposed to be "poweroff" according to the Upgrade Helper - which is being treated as the "source of truth" in this case